plugins/slack_app/app/controllers/slack_app_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SlackAppController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
That's coming! The CSRF check needs to be skipped here because the Slack API isn't a logged-in user.